framework/include/base/FEProblem.h

```diff
@@ -1101,6 +1115,12 @@ class FEProblem :
   Adaptivity _adaptivity;
 #endif
 
+  /// Pointer to XFEM controller
+  XFEMInterface * _xfem;
+
+  ///Weight factors used by XFEM to modify qp weights for partial elements
+  std::map<dof_id_type, MooseArray<Real> > _xfem_weights;
```

Comment:
Whoops!  It doesn't actually matter what it should be because I should have deleted it during a refactor.